src/usr/share/gpgmailer/gpgmailer.py

```diff
@@ -116,6 +120,45 @@ def start_monitoring(self):
                                   str(exception))
                 self.logger.error(traceback.format_exc())
 
+    def _read_and_send_message(self, file_name, loop_start_time):
+        """Attempt to build a message and send it while handling exceptions for individual
+        messages.
```

Comment:
I'd split this into two separate sentences, "Attempts to build a message and send it. Handle all exceptions so that no one problematic message holds up the processing of other messages." Note that "attempts" is plural.

Reply:
"Attempts" is transitive, but "handle" should indeed be plural.